contracts/rbn-staking/Redeemer.sol

```diff
@@ -100,6 +101,27 @@ contract Redeemer is Ownable {
     emit RBNRedeemed(amountToRedeem);
   }
 
+  /**
+   * @dev Redeems the rbn using manual amount
+   * @param _amount is the amount
+   */
+  function redeemRBN(uint256 _amount) external onlyOwner {
+    require(
+      address(votingEscrowContract) != address(0),
+      "votingEscrowContract is 0x0"
+    );
```

Comment:
We don't have to check whether the`votingEscrowContract` is not address(0) as long as we check in the functions which set `votingEscrowContract`

Reply:
initially it will be 0, but I get your point we won't make this mistake of calling it if its 0.